### ISSUE TITLE
Add argument to subsetMaf to keep factor levels after query.

### ISSUE THREAD

**Issue by ggrimes** [PoisonAlien/maftools#186]
Can you please add an argument to the subsetMaf function to keep all factor levels, after subset.
I.e subsetMaf(mafobj,...,droplevels=FALSE) 

Thanks

Graeme

**Comment by PoisonAlien** [comment]
Hi,
Sorry for the delay. Most of the functions depend on exact factor levels, missing levels might lead to an issue. But I will make it as an argument soon and let you know. 
Please let me know if you get any errors.

**Comment by PoisonAlien** [comment]
Hi,
I have pushed a commit to include `droplevels` in subsetMaf. Let me know if it works for you. 
Install from GitHub.

**Comment by PoisonAlien** [comment]
Yes, that is the case. Other functions depend on exact factor levels and missing values cause issue. 
It will take bit of work to fix them all. But may I ask why would you need it in MAF object ?

**Comment by ggrimes** [comment]
I would like to see the percentage of samples with mutations in specific genes as part of a cohort.
If an individual has no mutations (within a gene panel) then they would not be included in the calculation.

**Comment by PoisonAlien** [comment]
Hmm.. I have updated `subsetMaf` to apply for MAF object as well. Not sure how it will help you. Let me know if you have any errors.
